src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -111,9 +111,6 @@ function onOpen(isOpen: boolean) {
 
   &__label {
     @apply tw-text-gray-3;
-    @apply tw-truncate;
```

Comment:
Just one thing, I don't think we should allow the text from the filters to wrap, it should be in the same line

![image](https://github.com/Ebury/chameleon/assets/77344353/ecd742c0-32a0-414b-9265-da0496fca490)

Reply:
I don't think we should restrict it. It's unnatural to don't allow the element to wrap, fold etc. 
If you don't allow them to do it and you also remove the ellipsis,  then will be even worst they will overflow.

If we see that they wrap then we need to do something in the UI like remove a filter or so.

Reply:
Fine by me, but I would ask Alba anyways just in case...

Reply:
I asked Alba if she still wants to remove the ellipsis (which i think is the correct way to do this)

Reply:
We will go with this solution, if we don't like we can revert back to ellipsis or do some more major redesign of the header.